Source/Core/DolphinQt/QtUtils/DoubleClickEventFilter.h

```diff
@@ -9,6 +9,9 @@
 class DoubleClickEventFilter : public QObject
 {
   Q_OBJECT
+public:
+  DoubleClickEventFilter(QObject* parent);
```

Comment:
```suggestion
  explicit DoubleClickEventFilter(QObject* parent);
```

Reply:
Is there a -W flag so I stop forgetting this? :/ :P

Reply:
Yeah, unfortunately C++ didn't opt for `explicit` by default and require doing the inverse (adding `implicit`) instead =/

While there isn't a compiler flag for it, there is a clang-tidy [option](https://clang.llvm.org/extra/clang-tidy/checks/google-explicit-constructor.html) for it